src/test/java/com/exasol/adapter/dialects/exasol/ExasolSqlDialectLocalConnectionIT.java

```diff
@@ -19,6 +19,10 @@
  * <ul>
  */
 class ExasolSqlDialectLocalConnectionIT extends AbstractExasolSqlDialectIT {
+    static {
```

Comment:
This method of injection is unclean since its not clear where this property is consumed.
Using a protected method would be better.

Reply:
If I use a protected method, it will force to implement it in all 3 integration tests, but we only need it in one. So two of them will be empty. Is it okay?

Reply:
:heavy_check_mark: